spec/connect/errors_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+require 'ostruct'
+
+describe SUSE::Connect::ApiError do
+  let(:with_http_message) { OpenStruct.new(http_message: 'foo', code: 1337) }
+
+  let(:with_error) do
+    OpenStruct.new(
+      http_message: 'foo',
+      body: { 'error' => 'bar' }
+    )
+  end
+
+  let(:with_localized_error) do
+    OpenStruct.new(
+      http_message: 'foo',
+      body: { 'error' => 'bar',
+              'localized_error' => 'baz' }
+    )
+  end
+
+  describe '.code' do
```

Comment:
I think you meant `#code`.
This is just a convention, but the convention is to use `#` for instance methods, and `.` for class methods.